packages/backend/src/peripherals/database/shared/BaseRepository.ts

```diff
@@ -62,16 +62,22 @@ export type CheckConvention<T extends BaseRepository> = {
       * Arguments: any
       * Return type: count of deleted records  
 */
+const repositoryHistogram = new Histogram({
+  name: 'repository_method_duration_seconds',
+  help: 'duration histogram of repository methods',
+  labelNames: ['repository', 'method'],
+})
+
 export abstract class BaseRepository {
-  protected histogram: RepositoryHistogram
+  protected histogram: Histogram
```

Comment:
This RepositoryHistogram type is better, because it will not allow labels that are not in `labelNames` on initialization.